help/include/configure-topic-notifications.md

```diff
@@ -0,0 +1,22 @@
+{start_tabs}
+
+{tab|desktop-web}
+
+{!topic-actions.md!}
+
+1. Configure topic notifications using the row of icons at the top of the menu.
+
+!!! tip ""
+
+    You can also configure notifications by clicking the topic notifications icon
+    in the message recipient bar or in **Recent conversations**.
+
+{tab|mobile}
+
+{!topic-long-press-menu.md!}
+
+1. Tap **Mute topic** or **Unmute topic**.
+
+{!topic-long-press-menu-tip.md!}
```

Comment:
Maybe we should keep the mobile tab instructions only in **Mute or unmute a topic** since this might lead to potential confusion like unintentionally unfollowing a topic.

Reply:
I've updated the PR not to use this for now, yeah. @timabbott can decide if he's OK with the commit that moves the content out to the /include/ directory; it'll be useful (with some tweaks) whenever we add mobile support for following topics.